panel/models/perspective.ts

```diff
@@ -170,10 +167,12 @@ export class PerspectiveView extends HTMLBoxView {
     })
     container.innerHTML = "<perspective-viewer style='height:100%; width:100%;'></perspective-viewer>"
     this.perspective_element = container.children[0]
-    this.perspective_element.resetThemes([...Object.values(THEMES)]).catch(() => {})
-    if (this.model.toggle_config) {
-      this.perspective_element.toggleConfig()
-    }
+
+    const themesArray = Object.values(THEMES);
```

Comment:
Perspective uses the first theme in the array as the default theme. If we don't put the `model.theme` first in the list we will see a *glitch* when it later changes to the `model.theme`.